lib/ui/painting.dart

```diff
@@ -2956,11 +2965,11 @@ class Canvas extends NativeFieldWrapperClass2 {
   /// multiple draw commands intersect with the clip boundary, this can result
   /// in incorrect blending at the clip boundary. See [saveLayer] for a
   /// discussion of how to address that and some examples of using [clipRRect].
-  void clipRRect(RRect rrect) {
+  void clipRRect(RRect rrect, [bool doAntiAlias = true]) {
```

Comment:
please always use named arguments for booleans.
if this should be an optional argument (e.g. for performance reasons), please make it an enum.
also, make sure to assert non-null.